package-lock.json

```diff
@@ -2935,9 +2935,9 @@
       "dev": true
     },
     "@types/node": {
-      "version": "13.1.7",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-13.1.7.tgz",
-      "integrity": "sha512-HU0q9GXazqiKwviVxg9SI/+t/nAsGkvLDkIdxz+ObejG2nX6Si00TeLqHMoS+a/1tjH7a8YpKVQwtgHuMQsldg==",
+      "version": "12.12.25",
```

Comment:
We did we downgrade? We were on v 13 and now 12?

Reply:
The type annotation were creating issues with electron because they are using the version I changed it too. There's already a PR to update electron to a more recent type package , but the maintainers have not merged it yet.

In the future you can just look in package.json and avoid package-lock its just a lock file that represents package.json.

Reply:
Alright, sound good.